docs/wiki/user/installation.md

```diff
@@ -20,6 +20,11 @@ title: Installation
 - ACRE2 will try to copy the plugins to your TeamSpeak 3 installation directory. A pop-up will appear describing what the process did.
 - Launch TeamSpeak 3 and enable the ACRE2 plugin in the `Settings -> Plugins` window.
 
+### Configuration
+You can configure ACRE2 features to your needs. ACRE2 integrates fully into the [CBA_A3 Settings System](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System), accessible via the ingame configuration. This applies to [server side config](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#server-settings) as well to the client side.
```

Comment:
```suggestion
You can configure ACRE2 features to your needs. ACRE2 use [CBA Settings System](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System), accessible via the in-game configuration. This applies to [server settings](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#server-settings) as well as client settings.
```

Reply:
Should be `ACRE2 uses the [CBA`, much more natural